playbooks/check_fips_status.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Check FIPS status
+  hosts: "{{ edpm_override_hosts | default('all', true) }}"
+  strategy: linear
+  become: true
```

Comment:
can we avoid adding playbook level become: true
we should try to only elevate at the task level

Reply:
yeah, I agree. I used other playbooks as template, but i guess we should drop privileges from them as well.